app/api/ada/lib/storage/database/uncategorized/tables.js

```diff
@@ -0,0 +1,72 @@
+// @flow
+
+import { Type } from 'lovefield';
+import type { lf$schema$Builder } from 'lovefield';
+
+
+export type KeyInsert = {|
+  Hash: string,
+  IsEncrypted: boolean,
+  PasswordLastUpdate: Date | null,
+|};
+export type KeyRow = {|
+  KeyId: number,
+  ...KeyInsert,
+|};
+export const KeySchema: {
+  +name: 'Key',
+  properties: $ObjMapi<KeyRow, ToSchemaProp>
+} = {
+  name: 'Key',
+  properties: {
+    KeyId: 'KeyId',
+    Hash: 'Hash',
+    IsEncrypted: 'IsEncrypted',
+    PasswordLastUpdate: 'PasswordLastUpdate',
+  }
+};
+
+export type ConceptualWalletInsert = {|
+  CoinType: number,
+  Name: string,
+  // NetworkId: number, // TODO
+|};
+export type ConceptualWalletRow = {|
+  ConceptualWalletId: number,
+  ...ConceptualWalletInsert,
+|};
+export const ConceptualWalletSchema: {
+  +name: 'ConceptualWallet',
+  properties: $ObjMapi<ConceptualWalletRow, ToSchemaProp>
+} = {
+  name: 'ConceptualWallet',
+  properties: {
+    ConceptualWalletId: 'ConceptualWalletId',
+    CoinType: 'CoinType',
+    Name: 'Name',
+  }
+};
+
+export const populateUncategorizedDb = (schemaBuilder: lf$schema$Builder) => {
+  schemaBuilder.createTable(ConceptualWalletSchema.name)
+    .addColumn(ConceptualWalletSchema.properties.ConceptualWalletId, Type.INTEGER)
+    .addColumn(ConceptualWalletSchema.properties.CoinType, Type.INTEGER)
+    .addColumn(ConceptualWalletSchema.properties.Name, Type.STRING)
+    .addPrimaryKey(
+      ([ConceptualWalletSchema.properties.ConceptualWalletId]: Array<string>),
```

Comment:
I didn’t get why this is `Array<string>` rather than `Type.INTEGER`

Reply:
https://github.com/google/lovefield/blob/8e47538d5f32986596a9e97ec97350cc6ed9ec1a/lib/schema/table_builder.js#L194-L225